src/chapters/arrays/multi-dimensional-arrays.rst

```diff
@@ -0,0 +1,107 @@
+Multi-Dimensional Arrays
+========================
+
+Earlier we learned that arrays can store any type of value. If that is true, can we store arrays inside
+of arrays? Well yes we can.... Get ready to venture into a multi-dimensional universe.
+
+.. index:: ! multi-dimensional array
+
+A **multi-dimensional array** is an array of arrays, meaning that the values inside the array are also arrays.
+The *inner* arrays can store other values such as strings, numbers, or even more arrays.
+
+.. figure:: figures/multi-dimensional-array-example.jpg
+   :alt: A label, synonyms, pointing to an array that contains arrays at it's four indexes. Each inner array has a list of words that are synonyms.
+
+Two Dimensional Arrays
+----------------------
+The simplest form of a multi-dimensional array is a two dimensional array. A two dimensional array is like a
+spreadsheet with rows and columns. To access items in a two dimensional array use square bracket notation and
+two indexes ``array[0][0]``. The first index is for the outer array, or the "row", and second index is for the inner array,
+or the "column".
+
+.. note:: The row and column analogy is used to help visualize a two dimensional array, however it's not a perfect analogy because there are no language rules forcing the inner arrays to all have the same length. The inner arrays are seperate arrays, possibly with varying lengths, that can be altered independently.
+
+.. admonition:: Example
+
+   Use a two dimensional array to contain three different lists of space shuttle crews.
+
+   .. sourcecode:: js
+
+      const shuttleCrews = [
+         ['Robert Gibson', 'Mark Lee', 'Mae Jemison'],
+         ['Kent Rominger', 'Ellen Ochoa', 'Bernard Harris'],
+         ['Eilen Collins', 'Winston Scott',  'Catherin Coleman']
+      ];
+
+      console.log(shuttleCrews[0][2]);
+      console.log(shuttleCrews[1][1]);
+      console.log(shuttleCrews[2][1]);
+
+   **Output**
+
+   ::
+
+      Mae Jemison
+      Ellen Ochoa
+      Winston Scott
+
+Multi-Dimensions and Array Methods
```

Comment:
If we are going to apply methods to 2-D arrays, then we need the methods page(s) to come before this one.

Reply:
moved 2D arrays to after methods